.devcontainer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update --fix-missing && \ 
+apt-get -y upgrade && \
+apt-get -y install git && \
+apt-get install -y --no-install-recommends build-essential software-properties-common python-is-python3 python3-pip python3-dev && \
+apt-get clean && \
+rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /app
+
+COPY . /app
+
+RUN python -m pip install -U pip
+
+RUN pip install -r /app/requirements.txt
+
+RUN python -m ipykernel install --user --name python38myenv
+
+# docker build . -t [your_project:container_tag] -p 8888:8888
```

Comment:
I guess these can be removed?

Reply:
Papermill requires a named kernel to run the workflows (the default kernel is the one previously used, but this may change between users. I find it best to name a separate kernel for running the workflow.